lib/active_model/introspector/lookup.rb

```diff
@@ -0,0 +1,16 @@
+module ActiveModel
```

Comment:
So I think if this is part of the FDL generator, then it should be in that namespace, and can be removed when all of that is removed.

If it's going to hang around afterwards, as @rgarner has mentioned in DMs, then I'd like to have a better explanation of what is for, as I'm not entirely following what use it has outside of our FDL generator.

Reply:
The main use it *would* have is generating a visual Ruby representation of our FDL anonymous classes for debugging purposes. By which I mean this namespace; this file will go away.

Reply:
The main problem you'll have in support is introspecting those anonymous classes; ActiveModel by itself doesn't make it easy to see metadata about attributes (the thing that has the `AttributeSet` with all its types and validators on is in an `@attributes` field with no accessor).

Reply:
The introspector can help here in a Rails console, but you probably also want something like `rake fdl:visualise[RM3787]` to print what the actual Ruby would be. You may of course survive without it!